makefile

```diff
@@ -1,8 +1,8 @@
 #
 # System configuration
 #
-PYTHON=/usr/bin/python3
-SHELLCHECK=/usr/bin/shellcheck
+PYTHON=/Library/Frameworks/Python.framework/Versions/3.6/bin/python3
```

Comment:
These values are used in the CI build on Travis and for any developer using Ubuntu so they shouldn't be changed. You can overwrite the values by invoking make with extra arguments like so: `make -e PYTHON=/Library/Frameworks/Python.framework/Versions/3.6/bin/python3 ci`.

Alternatively, could you please check if there's a cross-platform way of determining the path to Python and Shellcheck? E.g. the [which](https://linux.die.net/man/1/which) command would work on Linux but I'm not sure if this also works on Mac. To bind a value in the makefile to the result of a shell command, you should be able to use this syntax: `PYTHON := $(shell which python)`.

Reply:
Ok, noted! Thanks for the tip.

Reply:
which works Mac as well!